chia/cmds/start_funcs.py

```diff
@@ -37,7 +37,7 @@ async def create_start_daemon_connection(root_path: Path) -> Optional[DaemonProx
         if await connection.is_keyring_locked():
             passphrase = Keychain.get_cached_master_passphrase()
             if not Keychain.master_passphrase_is_valid(passphrase):
-                passphrase = get_current_passphrase()
+                passphrase = await asyncio.get_running_loop().run_in_executor(None, get_current_passphrase)
```

Comment:
default executor will create `num_cpus * 5` worker threads so I'd define a custom executor:
```
with ThreadPoolExecutor(1, "getpass") as executor:
        return await asyncio.get_event_loop().run_in_executor(executor, get_current_passphrase)

```

Reply:
I think the question would be whether we want to have a general pool for other potential uses like this or not.

Reply:
Updated to use a ThreadPoolExecutor with 1 worker thread.